behave_django/environment.py

```diff
@@ -109,19 +109,21 @@ def setup_test(self, context):
         This method runs the code necessary to create the test database, start
         the live server, etc.
         """
-        if django.VERSION >= (5, 2):
-            context.test.__class__._pre_setup(run=True)
-        else:
-            context.test._pre_setup(run=True)
-        context.test.setUpClass()
+        #if django.VERSION >= (5, 2):
+        #    context.test.__class__._pre_setup(run=True)
+        #else:
+        #    context.test._pre_setup(run=True)
+        #context.test.setUpClass()
         context.test()
 
     def teardown_test(self, context):
         """
         Tears down the Django test
         """
-        context.test.tearDownClass()
-        context.test._post_teardown(run=True)
+        #context.test.tearDownClass()
+        #context.test._post_teardown(run=True)
+        #if context.test.doClassCleanups:
+        #    context.test.doClassCleanups()
```

Comment:
As discussed in #164, instead of adding the cleanup code you should probably better make sure `addClassCleanup` is never executed to create cleanup tasks.

This is typically done by overriding the related class method by an empty implementation ([see example](https://github.com/paul-ri/behave-django/blob/main/behave_django/testcase.py#L63-L66)).